kibbeh/src/modules/keyboard-shortcuts/ChatKeybind.tsx

```diff
@@ -39,7 +40,7 @@ export const ChatKeybind: React.FC<ChatKeybindProps> = ({ className }) => {
       <div className={`ml-4`}>
         toggle chat keybind:{" "}
         <span className={`font-bold text-lg`}>
-          {active ? "listening" : CHAT}
+          {active ? "listening" : <VisualKeybind keybind={CHAT.toString()} />}
```

Comment:
why are all of these calling `.toString()`?  doesn't appear they will ever be returned as anything but a string

Reply:
Please see my [comment](https://github.com/benawad/dogehouse/pull/1935#issuecomment-817143927) below.